### PR TITLE
Upgrade 2.3

### PR DESCRIPTION
**Pull request by metinavc1**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,3 +15,14 @@ build/
 # IntelliJ
 *.iml
 .idea/
+acmepos/.settings/
```
Comment (gouline) [inline .gitignore]: All of these are redundant, just include `.classpath/`, `.settings/`, `.project` and `bin/`.

File: kebabpos/src/main/java/com/assemblypayments/kebabpos/Pos.java [hunk 2]
```diff
@@ -698,12 +706,50 @@ private boolean processInput(String[] spInput) {
...
+                printStatusAndActions();
+                System.out.print("> ");
+                break;
+               
+            case "receipt_header":
+            	String inputHeader = spInput[1].replace("\\r\\n", "\r\n");
+            	inputHeader = inputHeader.replace("\\\\", "\\");
+            	options.setCustomerReceiptHeader(inputHeader);
+            	options.setMerchantReceiptHeader(inputHeader);
+                SystemHelper.clearConsole();
```
Comment (gouline) [inline kebabpos/src/main/java/com/assemblypayments/kebabpos/Pos.java]: Please reformat this file with IntelliJ, as before.

**Review by gouline — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (metinavc1) [reply .gitignore]: fixed

File: kebabpos/src/main/java/com/assemblypayments/kebabpos/Pos.java (hunk 2 above)
Comment (metinavc1) [reply kebabpos/src/main/java/com/assemblypayments/kebabpos/Pos.java]: fixed

File: kebabpos/src/main/java/com/assemblypayments/kebabpos/Pos.java [hunk 3]
```diff
@@ -48,6 +49,9 @@ private void start(String[] args) {
         try {
             // This is how you instantiate SPI while checking for JDK compatibility.
             spi = new Spi(posId, eftposAddress, spiSecrets); // It is ok to not have the secrets yet to start with.
+            spi.posVendorId = "assembly";
```
Comment (gouline) [inline kebabpos/src/main/java/com/assemblypayments/kebabpos/Pos.java]: This won't work with the latest code review changes in the library, you need to call `setPosInfo("assembly", "2.3.0");`

**Review by gouline — CHANGES_REQUESTED** [review]

File: kebabpos/src/main/java/com/assemblypayments/kebabpos/Pos.java (hunk 3 above)
Comment (metinavc1) [reply kebabpos/src/main/java/com/assemblypayments/kebabpos/Pos.java]: Fixed

**Review by gouline — APPROVED** [review]

**Pull request merged**